modules/ROOT/pages/about-api-catalog-cli.adoc

```diff
@@ -0,0 +1,689 @@
+
+Custom trigger conditions enable you to further control what is published when the API Catalog CLI command runs. 
+
+This example shows `api-catalog publish-asset` command options that will publish only the APIs that are identified in the `triggerConditions` section of a descriptor file called `mydescriptor.yaml`.  
+
+*Command to publish:*
+
+----
+> api-catalog publish-asset -a https://anypoint.mulesoft.com -d mydescriptor.yaml -o 1234abc-b34a-7cd3-4s32-12345abc2345 -u myAnyPtAccount -p myPwd@4!myacct --trigger=ref:main --trigger=tag:migrate --trigger=status:complete
+
+----
+
+*Section in the specified descriptor file:*
+
+---
+ 
+ triggerConditions:
+  ref:
+   - main
+  tags:
+   - migrate
+   - republish
+  status:
+   - complete
+   - ready
+
+---
+
+==== Example 5: Publish Using a Jenkins Script 
+
+This example shows a Jenkins script that runs the `api-catalog publish-asset` command.
+
+[source,java]
+----
+pipeline {
+    agent any
+    environment {
+        ANYPOINT_ORG_ID = '1234abc-b34a-7cd3-4s32-12345abc2345'
+        CATALOG_DESCRIPTOR = './descriptor.yaml' <1>
+     }   
+     stages {
+        stage('git checkout') {
+            steps {
+                git url: 'git@github.com:mygitlocation/api-catalog-cli.git'
+                // additional checkout tasks
+            }
+        }
+        stage('Build Artifacts') {    
+            steps {
+                 // building
+            }
+        }
+        stage('API Cataloging') {    
+            steps {
+                 withCredentials([
+                    usernamePassword(credentialsId: 'my-anypoint-creds',
+                    usernameVariable: 'ANYPOINT_USERNAME',
+                    passwordVariable: 'ANYPOINT_PASSWORD')
+                ]) { 
+                    sh 'api-catalog-cli publish-asset -d $CATALOG_DESCRIPTOR -o $ANYPOINT_ORG_ID -a https://anypoint.mulesoft.com/ --trigger=branch:main' <2>
+                }
+            }
+        }
+         stage('Deploy') {    
+            steps {
+                // Any deployment tasks to be performed, here.
+            }
+        }
+    }
+----
+<1> Defines the organization and descriptor file values
+<2> Runs the `api-catalog publish-asset` command to publish APIs found in the `main` Github branch
+
+== Authentication
+
+You can configure Anypoint CLI authentication with username and password, client ID and client secret, or a bearer token.
+At least one method is required.
+
+=== Username and Password
+
+If you don't log in using single sign-on (SSO), you can use your username and password to log into the CLI directly.
+
+[%header%autowidth.spread,cols="a,a"]
+|===
+| Argument | Description
+| `username` | Your Anypoint Platform username
+
+You can also pass this value using the environment variable `export ANYPOINT_USERNAME=<name>`.
+| `password` | Your Anypoint Platform password
+
+You can also pass this value using the environment variable `export ANYPOINT_PASSWORD=<pwd>`.
```

Comment:
ditto re. alternatives and conditions under which your audience might want or need to use them